tests/integration/cluster.go

```diff
@@ -1297,7 +1297,7 @@ func NewClusterV3(t testutil.TB, cfg *ClusterConfig) *ClusterV3 {
 	}
```

Comment:
similarly, the SkipTestIfShortMode was new in 3.5.x and can skip downstream integration tests unexpectedly

Reply:
PTAL